### PR TITLE
Add scenefx package

### PR DESCRIPTION
**Pull request by Atmois**
Add scenefx package, dependency for swayfx package.

### CODE REVIEW

File: anda/lib/scenefx/scenefx.spec [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+%{meson_install}
+
+%check
+%{meson_test}
+
+
+%files
+%license LICENSE
+%doc README.md
+%{_libdir}/lib%{name}.so.%{abi_ver}{,.*}
```
Comment (madonuko) [inline anda/lib/scenefx/scenefx.spec]:
```suggestion
%{_libdir}/lib%{name}.so.*
```

and probably remove `%global abi_ver 1`

File: anda/lib/scenefx/scenefx.spec [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+BuildRequires:  pkgconfig(hwdata)
+BuildRequires:  pkgconfig(libdrm) >= 2.4.114
+BuildRequires:  pkgconfig(pixman-1) >= 0.42.0
+BuildRequires:  pkgconfig(wayland-client)
+BuildRequires:  pkgconfig(wayland-protocols) >= 1.32
+BuildRequires:  pkgconfig(wayland-scanner)
+BuildRequires:  pkgconfig(wayland-server) >= 1.22
+
+
+Packager:       Atmois atmois@atmois.com
```
Comment (madonuko) [inline anda/lib/scenefx/scenefx.spec]:
```suggestion
Packager:       Atmois <atmois@atmois.com>
```

**Review by madonuko — COMMENTED** [review]
`update.rhai`:

```ts
rpm.global("scenefxVersion", gh("wlrfx/scenefx"));
```

But you could just factor `%scenefxVersion` into `%version` anyway; in that case:

```ts
rpm.version(gh("wlrfx/scenefx"));
```

Overall the spec file is mostly well made though, nice.

**Comment by Atmois** [comment]
@madonuko added the RHAI :)

**Review by madonuko — APPROVED** [review]

**Comment by raboneko** [comment]
## 💔 Some backports could not be created

| Status | Branch | Result |
|:------:|:------:|:------|
|❌|f39|An unhandled error occurred. Please see the logs for details|
|❌|f40|An unhandled error occurred. Please see the logs for details|
|✅|f41|[](https://github.com/terrapkg/packages/pull/2219)|
|❌|el10|An unhandled error occurred. Please see the logs for details|

### Manual backport
To create the backport manually run:
```
backport --pr 2204
```

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport) and see the [Github Action logs](https://github.com/terrapkg/packages/actions/runs/11256192963) for details

**Comment by raboneko** [comment]
## 💔 Some backports could not be created

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|f39|[](https://github.com/terrapkg/packages/pull/2217)|
|❌|f40|An unhandled error occurred. Please see the logs for details|
|❌|f41|An unhandled error occurred. Please see the logs for details|
|✅|el10|[](https://github.com/terrapkg/packages/pull/2220)|

### Manual backport
To create the backport manually run:
```
backport --pr 2204
```

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport) and see the [Github Action logs](https://github.com/terrapkg/packages/actions/runs/11256191589) for details

**Comment by raboneko** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|f40|[](https://github.com/terrapkg/packages/pull/2218)|
|✅|f41|[](https://github.com/terrapkg/packages/pull/2219)|
|✅|el10|[](https://github.com/terrapkg/packages/pull/2220)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport) and see the [Github Action logs](https://github.com/terrapkg/packages/actions/runs/11256193958) for details

**Comment by raboneko** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|f39|[](https://github.com/terrapkg/packages/pull/2217)|
|✅|f40|[](https://github.com/terrapkg/packages/pull/2218)|
|✅|f41|[](https://github.com/terrapkg/packages/pull/2219)|
|✅|el10|[](https://github.com/terrapkg/packages/pull/2220)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport) and see the [Github Action logs](https://github.com/terrapkg/packages/actions/runs/11256193004) for details

**Comment by raboneko** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|el10|[](https://github.com/terrapkg/packages/pull/2220)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport) and see the [Github Action logs](https://github.com/terrapkg/packages/actions/runs/11256194583) for details

**Comment by raboneko** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|f39|[](https://github.com/terrapkg/packages/pull/2217)|
|✅|f40|[](https://github.com/terrapkg/packages/pull/2218)|
|✅|f41|[](https://github.com/terrapkg/packages/pull/2219)|
|✅|el10|[](https://github.com/terrapkg/packages/pull/2220)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport) and see the [Github Action logs](https://github.com/terrapkg/packages/actions/runs/11256192956) for details

**Pull request merged**